gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Jul 16 02:18:59 IST 2020
+#Sun Dec 06 19:05:41 IST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```

Comment:
Revert the changes in this file

Reply:
Minimum supported Gradle version is 6.5. Current version is 6.1.1. (there's an error I received)

Reply:
reverted